### ISSUE TITLE
protected public tags

### ISSUE THREAD

**Issue by michag86** [nextcloud/server#15412]
**Is your feature request related to a problem? Please describe.**
There is no type of tag that can be created from the administrator, assigned by the user, but not editable by the user. If I create a tag for the retention app with name "delete after 7 days", everyone can rename this to "delete after 30 days" or "very important, do not delete!"

**Describe the solution you'd like**
Create a new tag type: protected

**Describe alternatives you've considered**
protect renaming of tags if they are assigned to retention rules or somthing like that

cc @rullzer @MorrisJobke

**Comment by MorrisJobke** [comment]
cc @nickvergessen

**Comment by nickvergessen** [comment]
yeah totally agreed. It is a problem because the tags try to cover too many use cases with one system. It's the same, that some people want tags to be displayed for everyone on autocomplete. Others only want them to show up when they already see it on any object

**Comment by szaimen** [comment]
I am going to close this since there doesn't seem to be a lot of interest (no upvotes) and no respond since over 2 years. Please reopen if you still want to implement this.

**Comment by michag86** [comment]
I still think this is an issue as explained in my first post. If you try to use retention in a large system with many users, this is a problem.
